src/EditorFeatures/Core.Wpf/Interactive/InteractiveEvaluator.cs

```diff
@@ -94,59 +117,48 @@ internal abstract class InteractiveEvaluator : IResettableInteractiveEvaluator
             Debug.Assert(responseFileName == null || responseFileName.IndexOfAny(new[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar }) == -1);
 
             _threadingContext = threadingContext;
+            _taskQueue = new TaskQueue(listener, TaskScheduler.Default);
+            _shutdownCancellationSource = new CancellationTokenSource();
             _contentType = contentType;
-            _responseFilePath = Path.Combine(GetDesktopHostDirectory(), responseFileName);
+            _responseFileName = responseFileName;
```

Comment:
_responseFileName is annotated as nullable but the assignment is of a non-nullable parameter that's also assigned?

Reply:
Fixed.